### PR TITLE
periodic test run login

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Fixes 

periodic test run login

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): test

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: integration-tests/support/commands/hooks.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ before(() => {
     JSON.stringify({ 'application-list-getting-started-modal': true }),
   );
 
-  if (Cypress.env('PR_CHECK') === true) {
+  if (Cypress.env('PR_CHECK') || Cypress.env('PERIODIC_RUN')) {
```
Comment (jrichter1) [inline integration-tests/support/commands/hooks.ts:20]: `Cypress.env('PR_CHECK')` is going to have a default value of `'false'`, which in this case is still going to produce a truthy result.
In this configuration it pretty much always acts as if it's a pr check.

Comment (subhashkhileri) [reply integration-tests/support/commands/hooks.ts:20]: when `Cypress.env('PERIODIC_RUN')` is `true` I want to execute this if block for ephemeral env to login

Comment (subhashkhileri) [reply integration-tests/support/commands/hooks.ts:20]: Update the `PR_CHECK` to boolean in cofig

**Pull request merged**